README.md

```diff
@@ -107,8 +107,9 @@ The router ensures efficient request distribution among backends. It supports:
 - Exporting observability metrics for each serving engine instance, including QPS, time-to-first-token (TTFT), number of pending/running/finished requests, and uptime
 - Automatic service discovery and fault tolerance by Kubernetes API
 - Multiple different routing algorithms
-  - Round-robin routing
-  - Session-ID based routing
+  - Round-robin routing: Distributes requests evenly across all endpoints
```

Comment:
can we have a diagram to illustrate this different methods

definitely I can do that for the team

Reply:
Sure thing, let's wait for @KuntaiDu to finish the prefix aware routing #239 first and I'll restart this one.